config/webhook/admission_webhook.yaml

```diff
@@ -30,7 +30,7 @@ kind: Service
 metadata:
   labels:
     name: gateway-api-webhook-server
-    version: 0.0.1
+    version: dev
```

Comment:
I'd vote to remove this since it seems likely that this will be difficult to maintain or at least easy to mess up.